samples/JustSaying.Sample.Restaurant.KitchenConsole/appsettings.json

```diff
@@ -0,0 +1,11 @@
+{
+  "AWSRegion": "eu-west-1",
+  "AWSServiceUrl": "http://localhost:4100",
+  "Logging": {
+    "LogLevel": {
```

Comment:
Should I be able to add `"EventLog": "Trace"` here and have the messages contents logged?

Reply:
I think so - at some point in v7 we need to rename that, because it's easy to confuse with the Windows Event Log.